tests/installation/bootloader.pm

```diff
@@ -79,8 +79,33 @@ sub run() {
     if ( get_var("NETBOOT") ) {
         send_key "f4";
         assert_screen "inst-instsourcemenu", 4;
-        send_key "ret";
-        assert_screen "inst-instsourcedialog", 4;
+
+        # Select a source of net installation for sle11
+        if ( check_var('DISTRI', "sle")) {
```

Comment:
Why is this is a 'SLE 11 only' thing? Wouldn't the same code also work for openSUSE and SLE 12?
Wherever possible please write tests that work for SLE 11, SLE 12, and openSUSE - writing version specific tests should be avoided if they can, and I think in this case they can.